label/label.go

```diff
@@ -38,6 +39,7 @@ var knownRelayerLabels = map[Relayer]struct{}{
 	Rly:    {},
 	Hermes: {},
 }
+var mu sync.Mutex
```

Comment:
Having a global mutex is not ideal is there an object you can embed this? Maybe @mark-rushakoff has a good pattern here and I could easily be wrong.